### PR TITLE
Fix python example in readme

### PR DESCRIPTION
**Pull request by marioba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,19 +9,15 @@
...
+client = sdk.Client(
     url="https://app.qfield.cloud/api/v1/",
     username="user1",
     password="pass1",
 )
 
-try:
-    projects = client.projects()
-except requests.HttpRequest:
-    print("Oops!")
```
Comment (suricactus) [inline README.md:24]: Why did you remove the error handling part?

Comment (marioba) [reply README.md:24]: To keep it simple.
And it also looks wrong. `requests.HttpRequest` doesn't look like an exception and actually it doesn't seem to exist at all.

If we want to show also the Exception handling we should do `except requests.exceptions.RequestException`. I can add that.

Comment (suricactus) [reply README.md:24]: Yeah, you are right, the class is wrong. Qould be nice to have it there
